third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/call_mutual_borrows.exp

```diff
@@ -15,36 +15,3 @@ error: cannot pass mutable reference in local, which is still borrowed, as funct
    │                 ----------  ^^^^^^^^^^ passed here
    │                 │
    │                 previous call result
-
```

Comment:
Ok, but what about the errors on line 14 above?  Why do we still have that error?